ktor-client/ktor-client-curl/desktop/interop/libcurl.def

```diff
@@ -3,13 +3,14 @@ package = libcurl
 headers = curl/curl.h
 headerFilter = curl/*
 
-# there is no libz by default installed on windows, so we need to include it statically
-staticLibraries.mingw = libcurl.a libssl.a libcrypto.a libz.a
+# libcurl uses native Schannel for TLS on Windows
+# There is no libz by default installed on Windows, so we need to include it statically
+staticLibraries.mingw = libcurl.a libnghttp2.a libzlib.a
 compilerOpts.mingw = -DCURL_STATICLIB
+linkerOpts.mingw = -liphlpapi -lsecur32 -lcrypt32 -lws2_32
 
-# there is no need to use openssl on macos, as curl is built using out-of-the-box security framework
-staticLibraries.osx = libcurl.a
-linkerOpts.osx = -framework Security -framework SystemConfiguration
+staticLibraries.osx = libcurl.a libnghttp2.a libssl.a libcrypto.a
```

Comment:
Is there a reason why we can't build `curl` using the macOS Security framework instead of OpenSSL?